src/app/league/models.py

```diff
@@ -10,6 +10,7 @@ class LeagueModel(BaseModel):
     name = ndb.StringProperty()
     rating_scheme = ndb.StringProperty(required=True, choices=['ELO', 'type1', 'type2'])
     description = ndb.TextProperty(indexed=False)
+    participant_count = ndb.IntegerProperty()
```

Comment:
set a default of 0 here like ``` ndb.IntegerProperty(default=0) ```
i know you have it on the create_league, but then this property is only set to a value when create_league is called. By contrast, if you set a default value on the property declaration, the value will be set the next time you call ``` .put() ``` on the entity, which could happen during other more common operations like update_league